app/processing/siti-agri/handle-known-defects/remove-defunct-participation-fund.js

```diff
@@ -1,8 +1,12 @@
-const { convertToPence } = require('../../currency-convert')
+const { convertToPence } = require('../../../currency-convert')
+const { sfiPilot } = require('../../../schemes')
 const GROSS_LINE_DESCRIPTION = 'G00 - Gross value of claim'
 const PARTICIPATION_PAYMENT_SCHEME_CODE = '80009'
```

Comment:
I'm wondering is these constants couldn't be made available for test objects -- sep. ticket if viable anyway